src/sim/model/ModelClass.py

```diff
@@ -22,10 +20,10 @@ class RobotModel:
     def __init__(self, ip, thread=None):
         self.ip = ip
         self.robotItem = None
-        self.detections = list()
         self.thread = thread
         
 
 class BlockerModel:
     def __init__(self, item):
-        self.blockerItem = item
\ No newline at end of file
+        self.blockerItem = item
+        
```

Comment:
why

Reply:
Python requires an additional line at the end of its files